Minecraft Central/presence.ts

```diff
@@ -0,0 +1,447 @@
+var presence = new Presence({
+    clientId: "693097839424831489",
+    mediaKeys: false
+});

```

Comment:
mediakeys has been fully removed, so please remove it here aswell :)

Reply:
Done.
Again, thanks for the help. :)